test/integration/roles/test_apache2_module/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- name: include only on supported systems
+  include: actualtest.yml
+  when: ansible_os_family in ['Debian', 'Suse']
```

Comment:
RHEL (at least centos7, which I tested) does not have a2enmod/a2dismod

As far as I understand debian came up with it and opensuse also implemented it, but independently and with other arguments.